src/platform/plugins/shared/dashboard/public/dashboard_api/unified_search_manager.ts

```diff
@@ -64,7 +64,6 @@ export function initializeUnifiedSearchManager(
     timefilter: { timefilter: timefilterService },
   } = dataService.query;
 
-  const filters$ = new BehaviorSubject<Filter[] | undefined>(undefined);
```

Comment:
I think we should publish `unifiedSearchFilters$` to children API as `filters$`, right?